oar/README.md

```diff
@@ -102,7 +102,7 @@ $ oar -r $release-version image-signed-check
 ```
 $ oar -r $release-version drop-bugs
 ```
-11. This command will change advisory status e.g. REL_PREP, and close QE related JIRA subtasks
+11. This command will change advisory status e.g. REL_PREP, and close QE related JIRA subtasks. Refresh api is triggered to check for secalerts on RHSA's. If it fails it will throw appropriate error message
```

Comment:
actually we don't need to mention which api is used. key point here is we need to let the users know, the cmd will check `blocking` secalert for RHSA advisory, if blocking alert is found, cmd will stop there and prompt the appropriate message